api/kyverno/v2alpha1/policy_exception_types.go

```diff
@@ -71,6 +71,14 @@ type PolicyExceptionSpec struct {
 
 	// Exceptions is a list policy/rules to be excluded
 	Exceptions []Exception `json:"exceptions"`
+
+	// Images is a list of images to check if a resource applies to the exception.
+	// Applicable only to rule types that apply to images: verifyImages, validate.podSecurity.
+	Images *[]string `json:"images,omitempty"`
```

Comment:
Why we declate it as `*[]string` instead of `[]string`?

Reply:
I kept it as pointer so that it can be `nil`. I believe we want to make this field as optional right?

Reply:
Yes this field is optional. You can use `[]string` instead.